crates/pink/pink-extension/src/lib.rs

```diff
@@ -183,8 +233,23 @@ impl PinkEvent {
     }
 }
 
-/// Turn on on_block_end feature and set it's selector
+/// Sets a hook receiver for a given hook point.
```

Comment:
We should mention it's a privileged operation

Reply:
Done